src/utils/api/lsk/transactions.js

```diff
@@ -0,0 +1,36 @@
+import { toRawLsk } from '../../../utils/lsk';
+import { getTimestampFromFirstBlock } from '../../datetime';
+import txFilters from '../../../constants/transactionFilters';
+
+// eslint-disable-next-line max-statements, complexity, import/prefer-default-export
+export const getTransactions = ({
+  apiClient, address, limit, offset, type = undefined,
+  sort = 'timestamp:desc', filter = txFilters.all, customFilters = {},
+}) => {
+  const params = {
+    limit,
+    offset,
+    sort,
+  };
+
+  if (type !== undefined) params.type = type;
```

Comment:
I think you can just put like
```suggestion
  if (type) params.type = type;
```

Reply:
I cannot, because send type value is `0` and it wouldn't set it https://github.com/LiskHQ/lisk-hub/blob/0425e51423a85feb8be5cf349f69d63e38fee62d/src/constants/transactionTypes.js#L2

Reply:
oh ok good point, then everything clear